src/pages/MainPage.tsx

```diff
@@ -1,61 +1,47 @@
-import { IonButton, IonButtons, IonContent, IonHeader,  IonIcon,  IonMenuButton,  IonModal,  IonPage, IonTitle, IonToolbar } from "@ionic/react"
-import { listSharp } from "ionicons/icons"
-import { useRef } from "react"
-import InventoryList from "../components/InventoryList"
+import {
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonLabel,
+  IonPage,
+  IonRouterOutlet,
+  IonTab,
+  IonTabBar,
+  IonTabButton,
+  IonTabs,
+  IonTitle,
+  IonToolbar,
+} from "@ionic/react";
+import { listSharp, map } from "ionicons/icons";
+import { Redirect, Route } from "react-router";
+import InventoryListPage from "./InventoryListPage";
+// import { Route } from "workbox-routing";
+import MapPage from "./MapPage";
 
-import MainMap from "../components/MainMapMaplibre"
-import { useSettings } from "../context/settings"
+const MainPage = () => {
```

Comment:
Also, I think MainPage is not the right name here, as I only use Page for components that actually contain a `<IonPage>` at top level.

This should be a MainTabs component. Maybe we even put it into the Navigation directly, or define the component in the Navigation.tsx file, because this component does not really work without the navigation around it. @JesJehle  What do you think?

Reply:
Ok, I added MainTabs to the Navigation module. Seems more clean.